README.md

```diff
@@ -278,36 +222,87 @@ Check out `tune --help` for all possible CLI commands and options. For more info
 
 &nbsp;
 
-## Design Principles
+---
+
+## Llama3 and Llama3.1
+
+torchtune supports fine-tuning for the Llama3/Llama3.1 8B, 70B, and 405B size models. You can fine-tune the 8B model with LoRA, QLoRA and full fine-tunes on one or more GPUs. You can also fine-tune the 70B model with QLoRA on a single device or LoRA and full-finetunes on multiple devices. Finally, you can fine-tune the 405B model on a single node with QLoRA. For all the details, take a look at our [tutorial](https://pytorch.org/torchtune/main/tutorials/llama3.html).
+
+> [!NOTE]
+> Our Llama3 and Llama3.1 LoRA and QLoRA configs default to the instruct fine-tuned models. This is because not all special token embeddings are initialized in the base 8B and 70B models.
+
+
+### Single GPU
+
+LoRA 8B
+
+```bash
+tune run lora_finetune_single_device --config llama3_1/8B_lora_single_device
+```
+
+QLoRA 8B
+
+```bash
+tune run lora_finetune_single_device --config llama3_1/8B_qlora_single_device
+```
+
+Full 8B
+
+```bash
+tune run full_finetune_single_device --config llama3_1/8B_full_single_device
+```
+
+### Multi GPU
+
+Full 8B
+
+```bash
+tune run --nproc_per_node 4 full_finetune_distributed --config llama3_1/8B_full
+```
+
+LoRA 8B
 
-torchtune embodies PyTorch’s design philosophy [[details](https://pytorch.org/docs/stable/community/design.html)], especially "usability over everything else".
+```bash
+tune run --nproc_per_node 2 lora_finetune_distributed --config llama3_1/8B_lora
+```
 
-### Native PyTorch
+LoRA 70B
+
+Note that the download command for the Meta-Llama3 70B model slightly differs from download commands for the 8B models. This is because we use the HuggingFace [safetensor](https://huggingface.co/docs/safetensors/en/index) model format to load the model. To download the 70B model, run
+```bash
+tune download meta-llama/Meta-Llama-3.1-70b --hf-token <> --output-dir /tmp/Meta-Llama-3.1-70b --ignore-patterns "original/consolidated*"
```

Comment:
Not sure if i fully follow why for llama 8b we just share tune run, but for 70b, we have also download instructions here.